evals/evaluation/autorag/evaluation/ragas_evaluation_benchmark.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
the directory `evaluation` is not needed? shall we put the `ragas_evaluation_benchmark.py` at the `autorag/` instead of `autorag/evaluation`. 

shall we change the file name as 'main.py' or others, like `run_autorag.py` ?

Reply:
ok